src/components/dls/Carousel/Carousel.tsx

```diff
@@ -0,0 +1,31 @@
+import { ReactNode } from 'react';
```

Comment:
Would be great if we have a Storybook for the Carousel since we will be reusing it in the future

Reply:
I agree, will do it in a separate PR